content/en/blog/_posts/2022-04-28-cni-version-related-breaking-change.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: "CNI version-related breaking change in Kubernetes 1.24 GA"
+date: 2022-04-28T12:00:00+0000
+slug: cni-version-related-breaking-change-k8s-1.24
+canonicalUrl: https://kubernetes.dev/blog/2022/04/28/cni-version-related-breaking-change-k8s-1.24/
+---
+
+**Authors & Contributors:** [Mike Brown](https://github.com/mikebrow), [Rolfe Dlugy-Hegwer](https://github.com/rolfedh),[Nate W.](https://github.com/nate-double-u)
+
+---
+
+**IMPORTANT** Urgent upgrade notes
+(No, really, you MUST read this before you upgrade) Before you upgrade to Kubernetes 1.24, if you use a CNI network plugin that complies with Container Network Interface (CNI) versions earlier than v1.0.0 and you plan to use any of the following container runtimes, you must specify the CNI version as described in [Declaring the CNI version of your plugin](TBD). Otherwise, your network will break.
+
+This issue applies to the following container runtimes:
+containerd v1.6.0 and later
+CRI-O 1.24 and later
+
+If you performed the upgrade without following these steps and your network is broken, see [<link to troubleshooting topic>]().
+
+FEATURE STATE: Kubernetes v1.24 GA
+
+TLDR... For Kubernetes v1.24 GA, when you upgrade one of the following container runtimes, safely drain (cordon) your node, upgrade your CNI plugins to the latest versions, and confirm that your CNI config files specify/match the correct versions for your plugins. For `containerd`, ensure that you have installed the latest version (v1.0.0 or later) of the CNI loopback plugin.
```

Comment:
```suggestion
TL;DR: For Kubernetes v1.24 GA, when you upgrade one of the following container runtimes, safely drain (cordon) your node, upgrade your CNI plugins to the latest versions, and confirm that your CNI config files specify/match the correct versions for your plugins. For `containerd`, ensure that you have installed the latest version (v1.0.0 or later) of the CNI loopback plugin.
```

Reply:
Sorry - this was WIP - I ended up consolidating this content.